application/src/main/java/org/togetherjava/tjbot/features/basic/MemberCounter.java

```diff
@@ -0,0 +1,48 @@
+package org.togetherjava.tjbot.features.basic;
+
+import net.dv8tion.jda.api.JDA;
+import net.dv8tion.jda.api.entities.Guild;
+import net.dv8tion.jda.api.entities.channel.concrete.Category;
+
+import org.togetherjava.tjbot.config.Config;
+import org.togetherjava.tjbot.features.Routine;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import static java.lang.Integer.parseInt;
+
+public class MemberCounter implements Routine {
```

Comment:
needs a javadoc

im not a fan of the name, id rename it to
MemberCountDisplayRoutine

Reply:
not done